hydra/test_utils/configs/ax_sweeper_plugin/banana.yaml

```diff
@@ -0,0 +1,32 @@
+defaults:
```

Comment:
Same.
Plugin specific code and configs does not belong in hydra-core.
Please move this to the plugin.

Reply:
Right :)